src/Microsoft.AspNetCore.Mvc.Core/Controllers/DefaultControllerActivator.cs

```diff
@@ -1,81 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System;
-using System.Reflection;
-using Microsoft.AspNetCore.Mvc.Core;
-using Microsoft.AspNetCore.Mvc.Infrastructure;
-using Microsoft.AspNetCore.Mvc.Internal;
-
-namespace Microsoft.AspNetCore.Mvc.Controllers
-{
-    /// <summary>
-    /// <see cref="IControllerActivator"/> that uses type activation to create controllers.
-    /// </summary>
-    public class DefaultControllerActivator : IControllerActivator
```

Comment:
1. Where's a bug tracking this work?
2. If we delete `DefaultControllerActivator`, what would the alternative be to anyone who used/derived from it?